__tests__/app.test.js

```diff
@@ -71,4 +71,45 @@ describe("GET: /api/articles/:article_id", () => {
             expect(response.body.msg).toBe('Invalid ID');
         })
     })
-})
\ No newline at end of file
+})
+
+describe('/api/teams/:article_id/comments', () => {
+    test('GET:200 sends an array of comments to the user by article_id', () => {
+      return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.comments).toEqual(expect.any(Array));
+          expect(Object.keys(response.body.comments[0])).toEqual(
+            expect.arrayContaining([
```

Comment:
This is ok I suppose however very much uneeded. You already have a forEach so why not just use your assertions in that ? eg. author: expect.any(String) etc